shared/app/push-prompt.native.tsx

```diff
@@ -13,73 +15,104 @@ type Props = {
       <Kb.WaitingButton
         fullWidth={true}
         onClick={props.onRequestPermissions}
-        label="Got it"
+        label="Allow notifications"
         waitingKey={Constants.permissionsRequestingWaitingKey}
         style={styles.button}
+        type="Success"
       />
-      <Kb.Button
-        type="Dim"
-        fullWidth={true}
-        onClick={props.onNoPermissions}
-        label="No thanks"
-        style={styles.button}
-      />
-      <Kb.Box style={styles.spacer} />
     </Kb.Box2>
-  </Kb.ScrollView>
+  </Kb.Box2>
 )
 
-const styles = Styles.styleSheetCreate({
-  button: {maxHeight: 40},
-  container: {padding: Styles.globalMargins.small},
-  image: Styles.platformStyles({
-    isMobile: {
-      height: 200,
-      width: '170%',
-    },
-  }),
-  scroll: {
-    backgroundColor: Styles.globalColors.white,
-    ...Styles.globalStyles.fillAbsolute,
-  },
-  scrollContent: {minHeight: '100%'},
-  spacer: {flexGrow: 1},
-  text: {
-    color: Styles.globalColors.black,
-  },
-})
-
 const mapStateToProps = () => ({})
 const mapDispatchToProps = dispatch => ({
   onNoPermissions: () => dispatch(PushGen.createRejectPermissions()),
   onRequestPermissions: () => dispatch(PushGen.createRequestPermissions()),
 })
 
+const PushPromptWithHeader = Container.withProps(
+  (props: Props & HeaderHocProps & ReturnType<typeof mapDispatchToProps>): Partial<HeaderHocProps> => ({
+    borderless: true,
+    customComponent: (
+      <Kb.Box2 direction="horizontal" fullWidth={true} centerChildren={true} style={styles.header}>
+        <Kb.Text type="BodyBig" negative={true}>
+          Allow notifications
+        </Kb.Text>
+        <Kb.ClickableBox onClick={props.onNoPermissions} style={styles.skip}>
+          <Kb.Text type="BodyBig" negative={true}>
+            Skip
+          </Kb.Text>
+        </Kb.ClickableBox>
+      </Kb.Box2>
+    ),
+    onRightAction: props.onNoPermissions,
+    rightActionLabel: 'Skip',
+  })
+)(HeaderHoc(PushPrompt))
+
+// TODO: uncomment when we make this a routed component
+// PushPrompt.navigationOptions = {
+//   header: null,
+//   headerTitle: 'Allow notifications',
+//   headerBottomStyle: {height: undefined},
+//   headerLeft: null, // no back button
+//   headerRightActions: () => (
+//     <Kb.Box2
+//       direction="horizontal"
+//       style={Styles.padding(Styles.globalMargins.tiny, Styles.globalMargins.tiny, 0)}
+//     >
+//       <Text type="BodyBig" negative={true}>
+//         Skip
+//       </Text>
+//     </Kb.Box2>
+//   ),
+// }
+
+const styles = Styles.styleSheetCreate(() => ({
+  button: {maxHeight: 40},
+  buttonContainer: {
+    flex: 1,
+    justifyContent: 'flex-end',
+  },
+  container: {
+    ...Styles.globalStyles.fillAbsolute,
+    backgroundColor: Styles.globalColors.blue,
+    padding: Styles.globalMargins.small,
+  },
+  header: {
+    backgroundColor: Styles.globalColors.blue,
+    height: '100%',
+    position: 'relative',
+  },
+  image: {
+    height: '55%',
```

Comment:
A little smaller and I think it'll be good. Flexbox might help but probably not worth fiddling with.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/11968340/63523161-98a50b80-c4c7-11e9-87b3-0e9d97d4e804.png">

Reply:
I'll get rid of the padding